libass/ass_directwrite.c

```diff
@@ -413,12 +448,17 @@ static void destroy_font(void *data)
 {
     FontPrivate *priv = (FontPrivate *) data;
 
-    IDWriteFont_Release(priv->font);
+    if (priv->font != NULL)
```

Comment:
I think simple `if (priv->font)` would be better.

Reply:
It would, I agree, but only if the lines immediately below followed that style.